pkg/config/config_template.yaml

```diff
@@ -3807,10 +3807,26 @@ api_key:
     ##                            Binds to 0.0.0.0 by default (accepting all packets).
     ##  * workers      - string - (Optional) Number of workers to use for this listener.
     ##                            Defaults to 1.
+    ##  * mapping      - (Optional) List of NetflowV9/IPFIX fields to additionally collect.
+    ##                              Defaults to None.
+    ##     * field       - integer - The Netflow field type to collect.
+    ##     * destination - string  - Name of the collected field, will be queryable under @<destination> in Datadog.
+    ##                              Default fields can be overridden, example `destination.port` will override
+    ##                              the default destination port collected.
+    ##     * type        - string  - The field type.
+    ##                              Available options are: string, integer, hex.
+    ##                              Defaults to hex.
```

Comment:
What about changing the order?

```
##     * type        - string  - The field type.
##                              Available options are: string, integer, hex.
##                              Defaults to hex.
##     * field       - integer - The Netflow field type to collect.
##     * destination - string  - Name of the collected field, will be queryable under @<destination> in Datadog.
##                              Default fields can be overridden, example `destination.port` will override
##                              the default destination port collected.
```

Since the `type` and `type` are directly related

Reply:
Oof very good point thank you!
The two `type` are not really related, the first one (Netflow field type) is an int identifying the field (e.g `aaa_username` -> field type 40000)
The second one is the data type of this field (for `aaa_username` this would be string)
Let me try to find a way to clarify this